### PR TITLE
Executive Copy - 2021-06-04

### PR DESCRIPTION
**Pull request by LongForWisdom**

### CODE REVIEW

File: governance/votes/Executive vote - June 4, 2021.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+**Core Unit Budget Distributions - June**
+- **40,000 DAI** will be distributed to **Real World Finance (RWF-001)**.
+- **100,500 DAI** will be distributed to **Risk (RISK-001)**. 
+- **80,000 DAI** will be distributed to **GovAlpha (GOV-001)**.
+- **510,000 DAI** will be distributed to **Protocol Engineering (PE-001)**.
+- **126,117 DAI** will be distributed to **Growth (GRO-001)**.
+- **44,375 DAI** will be distributed to **Content Production (MKT-001)**.
+- **642,135 DAI** will be distributed to **Sustainable Ecosystem Scaling (SES-001)**.
+
+**Liquidations 2.0**
```
Comment (prose11) [inline governance/votes/Executive vote - June 4, 2021.md:26]: Don't think it's necessarily worth changing, but did notice the vaults are listed in different order here and in the detail section. All there in both sections :+1:

File: governance/votes/Executive vote - June 4, 2021.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+- TUSD-A
+- USDT-A
+- PAXUSD-A
+- GUSD-A
+- PSM-USDC-A
+- UNIV2DAIUSDC-A
+
+### Increase Global Debt Ceiling
+
+Due to the recent issue in which the Global Debt Ceiling limited usage of the Maker Protocol for a number of days, the Protocol Engineering Core Unit has proposed increasing the Global Debt Ceiling in this forum [post](https://forum.makerdao.com/t/propose-to-raise-the-global-debt-ceiling/8516). The Global Debt Ceiling will be increased by 500 million DAI if this executive proposal passes.
```
Comment (prose11) [inline governance/votes/Executive vote - June 4, 2021.md:80]: I know we usually have a "from" to "to" format for these increases that wouldn't work with the Global DC

Comment (prose11) [reply governance/votes/Executive vote - June 4, 2021.md:80]: Would it be worth saying something like:

Typically, the Global Debt Ceiling is the sum of all available Debt Ceilings. To alleviate the recent issue, this measure proposes adding a fixed amount of 500M to the sum of the debt ceilings, effectively raising the Global Debt Ceiling by 500M DAI.

**Review by prose11 — CHANGES_REQUESTED** [review]
Links are the only real fixes that need to be done, everything else could probably be dismissed as style preference!

File: governance/votes/Executive vote - June 4, 2021.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+- **510,000 DAI** will be distributed to **[Protocol Engineering (PE-001)]()** core unit at **0xe2c16c308b843eD02B09156388Cb240cEd58C01c** as specified in their active [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP7)
+- **126,117 DAI** will be distributed to the **[Growth (GRO-001)](https://mips.makerdao.com/mips/details/MIP39c2SP4)**  core unit at **0x7800C137A645c07132886539217ce192b9F0528e**  as specified in their active [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP4).
+- **44,375 DAI** will be distributed to the **[Content Production (MKT-001)](https://mips.makerdao.com/mips/details/MIP39c2SP5)**  core unit at **0x6A0Ce7dBb43Fe537E3Fd0Be12dc1882393895237** as specified in their active [budget proposal](https://mips.makerdao.com/mips/details/MIP41c4SP5) and modified within this forum [post](https://forum.makerdao.com/t/adding-content-production-multisig/8428/5).
+- **642,135 DAI** will be distributed to **Sustainable Ecosystem Scaling (SES-001)** core unit auditors wallet at **0x87AcDD9208f73bFc9207e1f6F0fDE906bcA95cc6** as specified in their active [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP10).
```
[the remark this answers is not in the document]
Comment (LongForWisdom) [reply governance/votes/Executive vote - June 4, 2021.md:60]: Yeah, not sure what to do about this one.

**Review by prose11 — APPROVED** [review]
Looks good to me!

**Pull request merged**